### DISCUSSION TITLE
Latest version

### DISCUSSION THREAD

**Discussion by PepeOnzima** [TheLastBen/fast-stable-diffusion#1152] [General]
2 days ago I was able to generate a panoplie of my images in different poses.

With the latest version, no matter what prompt I give it (1.5) it draws me sitting in a chair like one of my input pictures. Even if I write "standing up" as a prompt, it still draws me sitting in the exact same chair as my input picture!!!

I tried 3 times cleaning everything on my Gdrive and restarting and it made absolutly no difference!

**Comment by TheLastBen** [comment]
Start with 450 unet steps, slowly add 100 at a time.

**Reply by PepeOnzima** [reply]
OK, I will try and report. If I remember correctly the previous version had 3000 steps.

**Reply by TheLastBen** [reply]
the new instructions say start with 650 and slowly add 100 to the unet (the txt encoder max is 350 during the whole training)

**Comment by vesperialuna** [comment]
i was wondering why we can't use the 3000 steps anymore. i got better results from that. the new one that requires to start at 650 and below had a more disappointing output. repeated several times with 150 steps but  little to no improvement.

**Reply by TheLastBen** [reply]
use 10 instance images, if you don't like the new settings, use the dropdown menu and select the lowest learning rate for the unet and increase the number of steps to 2000+

**Reply by vesperialuna** [reply]
okay will try that. thank you.
